src/test/java/org/broadinstitute/hellbender/utils/io/IOUtilsUnitTest.java

```diff
@@ -270,5 +267,20 @@ public void testUrlEncodeDecode(final String string, final String encoded) {
         Assert.assertEquals(string, IOUtils.urlDecode(encoded));
     }
 
+    @Test
+    public void testLoadLargeRuntimeResource() throws IOException {
+        final Resource largeResource = new Resource(Resource.LARGE_RESOURCES_PATH + "/largeResourceTest.txt", null);
+        final File resourceFile = IOUtils.writeTempResource(largeResource);
```

Comment:
Might want to extract a simple utility method for loading a resource and writing to a temp location (would return a File)